convert.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
```

Comment:
This file should have some comments describing an example use for the kicker and fpga